Sites/GD2/Project 1/index.html

```diff
@@ -0,0 +1,71 @@
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+    <link rel="stylesheet" href="Styles.css" </head> <body>
+
+    <h1>Yes, the Octopus Is Smart as Heck. But Why?</h1>
+
+    <h2>It has eight arms, three hearts — and a plan. Scientists aren’t sure how the cephalopods got to be so
+        intelligent.</h2>
+
+
+    <p1>By Carl Zimmer </p1>
```

Comment:
You probably just want a `p` element here; there is no `p1`:
```suggestion
    <p>By Carl Zimmer </p>
```